Android/app/src/main/java/com/imobile3/groovypayments/ui/main/MainDashboardActivity.java

```diff
@@ -89,16 +87,14 @@ private void handleDashboardButtonClick(@NonNull MainDashboardButton button) {
                 break;
 
             case Management:
-                LogHelper.writeWithTrace(Level.INFO, TAG, "Management area not implemented");
-                break;
-
             case TimeTracking:
-                LogHelper.writeWithTrace(Level.INFO, TAG, "Time Tracking area not implemented");
-                break;
-
```

Comment:
I was considering leaving the log messages in place for the non-**Placeholder** buttons, what do you guys think?

Reply:
Why do you think they need to be removed? I think they can stay.

Reply:
Hm... After further consideration, I concur! 👍